### ISSUE TITLE
Now unusable with Youtube - Is this project dead?

### ISSUE THREAD

**Issue by gitguys** [el1t/uBlock-Safari#150]
### Describe the issue

YouTube hangs Safari and even if I try to whitelist it OBO doesn't seem to stop the trouble until I literally turn it off entirely by unchecking it in the Safari preferences.

### One or more specific URLs where the issue occurs

www.youtube.com

- MacOS version: Sierra latest
- Safari version: 12.1
- uBlock Origin version: latest

**Comment by ghost** [comment]
Yeah it's dead, 0 work on code since 21 April 2018.

**Comment by Theothepurveyor** [comment]
dammit…. so what to use instead?

**Comment by ghost** [comment]
> dammit…. so what to use instead?

I tried AdGuard but it sucks (take a lot of CPU ressources, lag...), so I have no idea. Another issue was creating to discuss this: https://github.com/el1t/uBlock-Safari/issues/145 but nothing is good IMHO.

The best idea is that someone who have rights access to this repo submit it to https://www.codeshelter.co

**Comment by agret** [comment]
> dammit…. so what to use instead?


https://itunes.apple.com/au/app/ublock/id1385985095?mt=12

**Comment by gitguys** [comment]
agret, I've heard that's a scam.

https://www.reddit.com/r/ublock/comments/32mos6/ublock_vs_ublock_origin/

**Comment by thekiefs** [comment]
Anyone tried Ghostery? 

https://www.ghostery.com

**Comment by soloriojulian** [comment]
wipr from the app store isn't a full replacement, but it works a lot better than other's i've tried.

**Comment by kusuriurikun** [comment]
Unfortunately, due to the present moves in development of Safari (which effectively eliminate ANY company from writing an effective adblocker--even AdGuard is running into serious issues between blocking of content being _completely_ un-integratable with Javascript/CSS modification and a limit of 50,000 entries per _extension_ for blocklists--such that AdGuard in future versions is likely to have to be shipped in _multiple extensions_) it can no longer be said to be secure or securable.

My sad advice is going to be a variation of what I used to tell Windows users in the bad old days of MSIE:

- Safari can no longer be secured, and thus should not be trusted on the Internet save for two purposes: Apple-specific sites (like the App Store) or for the purpose of getting a securable browser.
- Securable browsers include Firefox, Chrome, and most browsers derived from them (including Opera and Brave which are based on Chrome's base code Chromium, and Waterfox which is a fork of Firefox).
- The FIRST thing to do is install uBlock Origin from the extension stores for these browsers, and enable lists as needed.
- AFTER this is done (for those who miss the keychain functionality in Safari)--[Go here to learn how to export your existing keychain](https://gist.github.com/rmondello/b933231b1fcc83a7db0b), and then either from the App Store or (only once you've installed uBlock Origin!) _from a secure browser_ do a search for "KeepassX" or "Lastpass" and install these, and then IMPORT the keychain you just exported.  Then install the specific plugins for your browser for your preferred password manager, and you'll have the same functionality (and more!) as you did with Safari.
- No, it's not going to be QUITE as pretty as Safari.  Deal.  Unfortunately, Apple now thinks that *pretty* and keeping their walled garden unsullied by *outsiders* is more important than basic network security in 2019.  That's on them, not you.
